* unset LIBS during FoX configure to break circular dependency from make.inc
--- qe-6.3/install/extlibs_makefile.orig	2018-07-11 17:08:30.432173000 +0800
+++ qe-6.3/install/extlibs_makefile	2018-07-11 17:08:52.197762373 +0800
@@ -89,7 +89,7 @@
         mkdir ../FoX; \
 	(gzip -dc ../archive/fox.tgz | (cd ../FoX; tar -xvf -)); \
 	cd ../FoX/fox/; export FC=$(F90); export FCFLAGS="$(FOX_FLAGS)"; \
-	./configure --prefix=$(TOPDIR)/FoX ;\
+	LIBS="" ./configure --prefix=$(TOPDIR)/FoX ;\
 	$(MAKE) install; cd ../; rm -fr fox;fi
 # ELPA
 libelpa : libelpa_$(ELPA_LIBS_SWITCH)
